.codegen/model.go.tmpl

```diff
@@ -15,6 +16,16 @@ type {{.PascalName}} struct {
     {{- range .Fields}}
     {{.Comment "    // " 80}}
     {{.PascalName}} {{if .IsOptionalObject}}*{{end}}{{template "type" .Entity}} `{{template "field-tag" . }}`{{end}}
+
+	ForceSendFields []string `json:"-"`
+}
+
+func (s *{{.PascalName}}) UnmarshalJSON(b []byte) error {
+	return marshal.Unmarshal(b, s)
+}
+
+func (s {{.PascalName}}) MarshalJSON() ([]byte, error) {
+	return marshal.Marshal(s)
```

Comment:
We only need to add a custom `MarshalJSON` with `ForceSendFields` for types that:

1. are used in requests
2. have optional primitive fields (int, bool, string, ..). 

This PR adds it for literally **every response and request type**, which hurts the UX and makes documentation less readable.

<img width="486" alt="image" src="https://github.com/databricks/databricks-sdk-go/assets/259697/200c7215-b49d-4896-acb0-5a77a6d76805">

`UnmarshalJSON` is not necessary, as it's used for decoding responses.

Reply:
(2) Removed from structs without fields of basic types. 

As for (1), it is not only requests, but also structs which are subfields of a request (at any level).
I will check if we can easily determine that.